utils/build/docker/java/spring-boot/src/main/java/com/datadoghq/system_tests/springboot/AppSecIast.java

```diff
@@ -274,6 +274,18 @@ String secureXPath(final ServletRequest request) {
         return "XPath secure";
     }
 
+    @GetMapping("/trust-boundary-violation/test_insecure")
+    public String trustBoundaryViolationInSecureSpringBoot(final HttpServletRequest request) {
+      String paramValue = request.getParameter("username");
+      request.getSession().putValue("name", paramValue);
+      return "Trust Boundary violation page";
+    }
+
+    @GetMapping("/trust-boundary-violation/test_secure")
+    public String trustBoundaryViolationSecureSpringBoot(final HttpServletRequest request) {
+      return "Trust Boundary violation page";
```

Comment:
You should put something in the session that is not tainted

Reply:
applied suggestion